<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-1SAT413" failures="1" tests="5" name="passengers-count-3" time="15.93" errors="0" timestamp="2023-12-29T23:31:22 IST">
  <testcase classname="com.automation.selenium_docker.flightreservation.FlightReservationTest" name="userRegistrationTest" time="5.562"/>
  <testcase classname="com.automation.selenium_docker.flightreservation.FlightReservationTest" name="registrationConfirmationTest" time="1.387"/>
  <testcase classname="com.automation.selenium_docker.flightreservation.FlightReservationTest" name="flightSearchTest" time="2.039"/>
  <testcase classname="com.automation.selenium_docker.flightreservation.FlightReservationTest" name="flightSelectionTest" time="3.363"/>
  <testcase classname="com.automation.selenium_docker.flightreservation.FlightReservationTest" name="flightReservationConfirmationTest" time="1.443">
    <failure type="java.lang.AssertionError" message="expected [$1763 USD] but found [$1753 USD]">
      <![CDATA[java.lang.AssertionError: expected [$1763 USD] but found [$1753 USD]
at com.automation.selenium_docker.flightreservation.FlightReservationTest.flightReservationConfirmationTest(FlightReservationTest.java:76)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- flightReservationConfirmationTest -->
</testsuite> <!-- passengers-count-3 -->
